torch_frame/data/multi_nested_tensor.py

```diff
@@ -199,6 +217,23 @@ def device(self) -> torch.device:
 
     def size(self, dim: int) -> int:
         r"""Dimension of the :class:`torch_frame.data.MultiNestedTensor`
+
+        Example:
+            >>> import torch
+            >>> tensor_mat = [
+            ...    [torch.tensor([1, 2]), torch.tensor([3])],
+            ...    [torch.tensor([4]), torch.tensor([5, 6, 7])],
+            ...    [torch.tensor([8, 9]), torch.tensor([10])],
```

Comment:
prolly only need size 2?

Reply:
Just wanted to use different size than `size(1)`